repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -233,25 +233,25 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     } else {
       val checkHeader = checkHeaders(requestedTenantId, requestedContactId)
       val userPermissions = getPermissions(checkHeader)
-      val authPermissions = authorizeDevice(userPermissions, requestedDeviceId)
-      val allPermissions =  authPermissions match {
+      val allPermissions = userPermissions match {
```

Comment:
we should turn this chunk of code into a method probably named getInventory, and rename the other